bigbluebutton-html5/imports/ui/components/chat/message-list/message-list-item/styles.scss

```diff
@@ -32,7 +32,7 @@
 }
 
 .avatarWrapper {
-  flex-basis: 1.65rem;
+  flex-basis: 2.25rem;
```

Comment:
This value stayed `1.65rem` for 2 years. Do you know what changed elsewhere that requires changing it to `2.25rem`?

Reply:
The addition of a transparent border around the avatar, it was introduced as part of the high contrast theme styling.

https://github.com/bigbluebutton/bigbluebutton/pull/7192/files#diff-a20bf143031c93cd7278a51e2d41dd45R23